src/core/dbus-unit.c

```diff
@@ -1620,7 +1620,7 @@ int bus_unit_queue_job(
 
         _cleanup_(sd_bus_message_unrefp) sd_bus_message *reply = NULL;
         _cleanup_free_ char *job_path = NULL, *unit_path = NULL;
-        _cleanup_(set_freep) Set *affected = NULL;
+        _cleanup_set_free_ Set *affected = NULL;
```

Comment:
i'd probably go the other way and get rid of `_cleanup_set_free_` tree wide, but doesn#t matter